src/main/java/pl/betoncraft/betonquest/notify/Notify.java

```diff
@@ -39,6 +39,7 @@
 /**
  * Create a short message
  */
+@SuppressWarnings("PMD.ClassNamingConventions")
```

Comment:
Will the Notify class also be reworked or why is this suppressed?

Reply:
`Notify` as a exclusively static class and thus has to be renamed to `NotifyUtil`, but even that would be an API breaking change.